### ISSUE TITLE
[Bug] [a11y] ETP toggle accessibility updates

### ISSUE THREAD

**Issue by softvision-miralobontiu** [mozilla-mobile/fenix#7720]
## Prerequisite
Make sure you have Google Accessibility Scanner installed.

## Steps to reproduce
1. Launch Fenix and go to a website (i.e. cnn.com).
2. Tap on the ETP shield icon.
3. Using the Accessibility Scanner scan the Tracking protection context menu.

### Expected behavior
No suggestions are made.

### Actual behavior
The suggestion is to increase the item's height from 27dp to 48dp or larger - as seen in the screenshot below.

### Device information
* Android device: Samsung Galaxy Tab S3 (Android 8), Samsung Galaxy Note 8 (Android 9)
* Fenix version: RC 3.0.2, Beta build 3.1.0-beta 3, Nightly build from 01/15

![etp_Accessibility Scanner](https://user-images.githubusercontent.com/40694158/72439591-e4768b00-37af-11ea-825f-d001315e3a5b.jpg)

**Comment by sblatz** [comment]
This should be one cohesive element titled "Enhanced tracking protection" with a role of switch and the state of on/off. The whole thing should be only one focusable item. The supplementary text should be "Protections are ON for this site."

Ideal Talkback: "Enhanced tracking protection ON, switch. Protections are ON for this site." 

Double tapping this one element would disable tracking protection.

**Comment by asavill** [comment]
Hey! Can I have crack at this :)?

**Comment by asavill** [comment]
@sblatz I created a PR for this :). Tried to work with youR ideal talkback criteria, but struggled with the ordering. Ended up with "Enhanced tracking protection ON, switch" (so the supplementary text is omitted).

**Comment by ekager** [comment]
Reopening for verification

**Comment by sv-sdeiac** [comment]
Hi, I can confirm that this issue is fixed using the latest version of Firefox Preview Nightly 3/31 #20910606.

Devices:
- Samsung Galaxy Note 8 (Android 9);
- Samsung Galaxy Tab S6 (Android 9);
- Google Pixel 3 (Android 10).

**Comment by eeejay** [comment]
When I navigate to the switch I get "Protections are ON for this site, Switch". From the discussion above, it looked like the plan was to for this to be labeled with the title?

**Comment by asavill** [comment]
@eeejay To avoid adding new strings it was decided that the subtitle would be used instead.
